### ISSUE TITLE
Example test fails after initial scaffolding

### ISSUE THREAD

**Issue by markjackson02** [blitz-js/blitz#1426]
### What is the problem?

I ran `blitz new myApp` to scaffold my project, and `blitz db migrate` to initialize the database. When I unskip the test in `index.test.tsx` I get the following error: 

```
$ npm run test

> myApp@1.0.0 test C:\Users\markj\workspace\myApp
> jest

 FAIL  app/pages/index.test.tsx
  × renders blitz documentation link (142 ms)

  ● renders blitz documentation link

    It looks like you are trying to use Blitz's useQuery to fetch from third-party APIs. To do that, import useQuery directly from "react-query"

      12 | const UserInfo = () => {
      13 |   const currentUser = useCurrentUser()
    > 14 |   const [logoutMutation] = useMutation(logout)
         |                            ^
      15 | 
      16 |   if (currentUser) {
      17 |     return (

      at validateQueryFn (node_modules/@blitzjs/core/src/utils/react-query-utils.ts:57:11)
      at sanitize (node_modules/@blitzjs/core/src/utils/react-query-utils.ts:71:3)
      at useMutation (node_modules/@blitzjs/core/src/use-mutation.ts:18:37)
      at UserInfo (app/pages/index.tsx:14:28)
      at renderWithHooks (node_modules/react-dom/cjs/react-dom.development.js:15667:18)
      at mountIndeterminateComponent (node_modules/react-dom/cjs/react-dom.development.js:18529:13)
      at beginWork (node_modules/react-dom/cjs/react-dom.development.js:20019:16)
      at HTMLUnknownElement.callCallback (node_modules/react-dom/cjs/react-dom.development.js:156:14)
      at innerInvokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:315:27)
      at invokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:266:3)
      at HTMLUnknownElementImpl._dispatch (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:212:11)
      at HTMLUnknownElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:87:17)
      at HTMLUnknownElement.dispatchEvent (node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:144:23)
      at Object.invokeGuardedCallbackDev (node_modules/react-dom/cjs/react-dom.development.js:205:16)
      at invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:260:31)
      at beginWork$1 (node_modules/react-dom/cjs/react-dom.development.js:25156:7)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:23890:12)
      at workLoopSync (node_modules/react-dom/cjs/react-dom.development.js:23820:5)
      at renderRootSync (node_modules/react-dom/cjs/react-dom.development.js:23784:7)
      at performSyncWorkOnRoot (node_modules/react-dom/cjs/react-dom.development.js:23353:20)
      at scheduleUpdateOnFiber (node_modules/react-dom/cjs/react-dom.development.js:22808:7)
      at updateContainer (node_modules/react-dom/cjs/react-dom.development.js:26437:3)
      at node_modules/react-dom/cjs/react-dom.development.js:26900:7
      at unbatchedUpdates (node_modules/react-dom/cjs/react-dom.development.js:23497:12)
      at legacyRenderSubtreeIntoContainer (node_modules/react-dom/cjs/react-dom.development.js:26899:5)
      at Object.render (node_modules/react-dom/cjs/react-dom.development.js:26982:10)
      at node_modules/@testing-library/react/dist/pure.js:99:25
      at batchedUpdates$1 (node_modules/react-dom/cjs/react-dom.development.js:23450:12)
      at act (node_modules/react-dom/cjs/react-dom-test-utils.development.js:961:14)
      at render (node_modules/@testing-library/react/dist/pure.js:95:26)
      at render (test/utils.tsx:37:10)
      at Object.<anonymous> (app/pages/index.test.tsx:23:25)

  console.error
    Error: Uncaught [Error: It looks like you are trying to use Blitz's useQuery to fetch from third-party APIs. To do that, import useQuery directly from "react-query"]
        at reportException (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\helpers\runtime-script-errors.js:62:24)
        at innerInvokeEventListeners (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:331:9)
        at invokeEventListeners (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:266:3)
        at HTMLUnknownElementImpl._dispatch (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:212:11)   
        at HTMLUnknownElementImpl.dispatchEvent (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:87:17)        at HTMLUnknownElement.dispatchEvent (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\generated\EventTarget.js:144:23)     
        at Object.invokeGuardedCallbackDev (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:205:16)
        at invokeGuardedCallback (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:260:31)
        at beginWork$1 (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:25156:7)
        at performUnitOfWork (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23890:12) Error: It looks like you are trying to use Blitz's useQuery to fetch from third-party APIs. To do that, import useQuery directly from "react-query"
        at validateQueryFn (C:\Users\markj\workspace\myApp\node_modules\@blitzjs\core\src\utils\react-query-utils.ts:57:11)
        at sanitize (C:\Users\markj\workspace\myApp\node_modules\@blitzjs\core\src\utils\react-query-utils.ts:71:3)
        at useMutation (C:\Users\markj\workspace\myApp\node_modules\@blitzjs\core\src\use-mutation.ts:18:37)
        at UserInfo (C:\Users\markj\workspace\myApp\app\pages\index.tsx:14:28)
        at renderWithHooks (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:15667:18)
        at mountIndeterminateComponent (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:18529:13)
        at beginWork (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:20019:16)
        at HTMLUnknownElement.callCallback (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:156:14)
        at innerInvokeEventListeners (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:315:27)
        at invokeEventListeners (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:266:3)
        at HTMLUnknownElementImpl._dispatch (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:212:11)   
        at HTMLUnknownElementImpl.dispatchEvent (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\events\EventTarget-impl.js:87:17)        at HTMLUnknownElement.dispatchEvent (C:\Users\markj\workspace\myApp\node_modules\jsdom\lib\jsdom\living\generated\EventTarget.js:144:23)     
        at Object.invokeGuardedCallbackDev (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:205:16)
        at invokeGuardedCallback (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:260:31)
        at beginWork$1 (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:25156:7)
        at performUnitOfWork (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23890:12)
        at workLoopSync (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23820:5)
        at renderRootSync (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23784:7)
        at performSyncWorkOnRoot (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23353:20)
        at scheduleUpdateOnFiber (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:22808:7)
        at updateContainer (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:26437:3)
        at C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:26900:7
        at unbatchedUpdates (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23497:12)
        at legacyRenderSubtreeIntoContainer (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:26899:5)
        at Object.render (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:26982:10)
        at C:\Users\markj\workspace\myApp\node_modules\@testing-library\react\dist\pure.js:99:25
        at batchedUpdates$1 (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom.development.js:23450:12)
        at act (C:\Users\markj\workspace\myApp\node_modules\react-dom\cjs\react-dom-test-utils.development.js:961:14)
        at render (C:\Users\markj\workspace\myApp\node_modules\@testing-library\react\dist\pure.js:95:26)
        at render (C:\Users\markj\workspace\myApp\test\utils.tsx:37:10)
        at Object.<anonymous> (C:\Users\markj\workspace\myApp\app\pages\index.test.tsx:23:25)
        at Object.asyncJestTest (C:\Users\markj\workspace\myApp\node_modules\jest-jasmine2\build\jasmineAsyncInstall.js:106:37)
        at C:\Users\markj\workspace\myApp\node_modules\jest-jasmine2\build\queueRunner.js:45:12
        at new Promise (<anonymous>)
        at mapper (C:\Users\markj\workspace\myApp\node_modules\jest-jasmine2\build\queueRunner.js:28:19)
        at C:\Users\markj\workspace\myApp\node_modules\jest-jasmine2\build\queueRunner.js:75:41

      at VirtualConsole.<anonymous> (node_modules/jsdom/lib/jsdom/virtual-console.js:29:45)
      at reportException (node_modules/jsdom/lib/jsdom/living/helpers/runtime-script-errors.js:66:28)
      at innerInvokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:331:9)
      at invokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:266:3)
      at HTMLUnknownElementImpl._dispatch (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:212:11)
      at HTMLUnknownElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:87:17)

  console.error
    The above error occurred in the <UserInfo> component:
    
        at UserInfo
        at Suspense
        at div
        at main
        at div
        at Home
        at wrapper
    
    Consider adding an error boundary to your tree to customize error handling behavior.
    Visit https://fb.me/react-error-boundaries to learn more about error boundaries.

      at logCapturedError (node_modules/react-dom/cjs/react-dom.development.js:21087:23)
      at update.callback (node_modules/react-dom/cjs/react-dom.development.js:21120:5)
      at callCallback (node_modules/react-dom/cjs/react-dom.development.js:12786:12)
      at commitUpdateQueue (node_modules/react-dom/cjs/react-dom.development.js:12807:9)
      at commitLifeCycles (node_modules/react-dom/cjs/react-dom.development.js:21733:11)
      at commitLayoutEffects (node_modules/react-dom/cjs/react-dom.development.js:24567:7)
      at HTMLUnknownElement.callCallback (node_modules/react-dom/cjs/react-dom.development.js:156:14)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        5.123 s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! myApp@1.0.0 test: `jest`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the myApp@1.0.0 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\markj\AppData\Roaming\npm-cache\_logs\2020-11-06T17_52_57_008Z-debug.log
```

### Steps to Reproduce

1. `blitz new myApp`
2. `blitz db migrate`
3. unskip test 
4. `npm run test`

### Versions

```
Windows 10 | win32-x64 | Node: v14.15.0

blitz: 0.24.3 (global)
blitz: 0.24.3 (local)

  Package manager: yarn
  System:
    OS: Windows 10 10.0.19042
    CPU: (16) x64 AMD Ryzen 7 4800HS with Radeon Graphics
    Memory: 3.23 GB / 15.42 GB
  Binaries:
    Node: 14.15.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  npmPackages:
    @prisma/cli: 2.10.2 => 2.10.2
    @prisma/client: 2.10.2 => 2.10.2
    blitz: 0.24.3 => 0.24.3
    react: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8
    react-dom: 0.0.0-experimental-7f28234f8 => 0.0.0-experimental-7f28234f8
    typescript: 4.0.5 => 4.0.5
```

### Other

![image](https://user-images.githubusercontent.com/35121685/98399249-1eab6480-2030-11eb-8c0c-0c0d4257860f.png)

npm exit logs: 
```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'test'
1 verbose cli ]
2 info using npm@6.14.6
3 info using node@v12.18.3
4 verbose run-script [ 'pretest', 'test', 'posttest' ]
5 info lifecycle myApp@1.0.0~pretest: myApp@1.0.0
6 info lifecycle myApp@1.0.0~test: myApp@1.0.0
7 verbose lifecycle myApp@1.0.0~test: unsafe-perm in lifecycle true
8 verbose lifecycle myApp@1.0.0~test: PATH: C:\Users\markj\AppData\Roaming\nvm\v12.18.3\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Users\markj\workspace\myApp\node_modules\.bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\markj\bin;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\Microsoft VS Code\bin;C:\Users\markj\AppData\Roaming\nvm;C:\Program Files\nodejs;C:\Program Files (x86)\Yarn\bin;C:\Program Files\PostgreSQL\11\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\WINDOWS\System32\OpenSSH;C:\Program Files\Git\cmd;C:\Program Files (x86)\Calibre2;C:\Program Files\UniConvertor-2.0rc4;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0;C:\Windows\System32\OpenSSH;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\Microsoft VS Code\bin;C:\Users\markj\AppData\Roaming\nvm;C:\Program Files\nodejs;C:\Program Files (x86)\Yarn\bin;C:\Program Files\PostgreSQL\11\bin;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\WINDOWS\System32\OpenSSH;C:\Program Files\Git\cmd;C:\Program Files (x86)\Calibre2;C:\Program Files\UniConvertor-2.0rc4;C:\Users\markj\AppData\Local\Programs\Python\Python38-32;C:\Users\markj\AppData\Local\Programs\Python\Python38-32\Scripts;C:\Program Files\PostgreSQL\11\bin;C:\Program Files - self installed\apache-maven-3.6.3\bin
9 verbose lifecycle myApp@1.0.0~test: CWD: C:\Users\markj\workspace\myApp
10 silly lifecycle myApp@1.0.0~test: Args: [ '/d /s /c', 'jest' ]
11 silly lifecycle myApp@1.0.0~test: Returned: code: 1  signal: null
12 info lifecycle myApp@1.0.0~test: Failed to exec test script
13 verbose stack Error: myApp@1.0.0 test: `jest`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Users\markj\AppData\Roaming\nvm\v12.18.3\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (C:\Users\markj\AppData\Roaming\nvm\v12.18.3\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid myApp@1.0.0
15 verbose cwd C:\Users\markj\workspace\myApp
16 verbose Windows_NT 10.0.19042
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "test"
18 verbose node v12.18.3
19 verbose npm  v6.14.6
20 error code ELIFECYCLE
21 error errno 1
22 error myApp@1.0.0 test: `jest`
22 error Exit status 1
23 error Failed at the myApp@1.0.0 test script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

**Comment by flybayer** [comment]
This should be fixed as of [`v0.25.0-canary.0`](https://github.com/blitz-js/blitz/releases). Can you check and verify?

**Comment by markjackson02** [comment]
Yep, after upgrading to `blitz@canary` the test is fixed. Thank you!
